content/blog/2024/how-to-write-stellar-proposals.md

```diff
@@ -0,0 +1,87 @@
+[AI Engineering in Python: System Design 101](https://in.pycon.org/cfp/pycon-india-2023/proposals/ai-engineering-in-python-system-design-101~elY5g/)
+
+2. Providing detailed descriptions <br />
+[Revealing Uncommon Tricks: 25 Obscure Pandas & NumPy Hacks learnt over 5 years of being a Data Scientist](https://in.pycon.org/cfp/pycon-india-2023/proposals/openai-whisper-and-its-amazing-power-to-do-fine-tuning-demonstrated-on-my-mother-tongue~eENWa/) <br />
+[The Why Conundrum - Practical causal inference and discovery with python](https://in.pycon.org/cfp/pycon-india-2023/proposals/the-why-conundrum-practical-causal-inference-and-discovery-with-python~eZ6YR/)
+
+3. Attaching slides/videos if available: <br />
+[Revealing Uncommon Tricks: 25 Obscure Pandas & NumPy Hacks learnt over 5 years of being a Data Scientist](https://in.pycon.org/cfp/pycon-india-2023/proposals/revealing-uncommon-tricks-25-obscure-pandas-numpy-hacks-learnt-over-5-years-of-being-a-data-scientist~bkR5N/) <br />
+[Taking a Multilingual Conversational Engine to Production: Theory to Reality](https://in.pycon.org/cfp/pycon-india-2023/proposals/taking-a-multilingual-conversational-engine-to-production-theory-to-reality~ejRQv/) <br />
+[AI Engineering in Python: System Design 101](https://in.pycon.org/cfp/pycon-india-2023/proposals/ai-engineering-in-python-system-design-101~elY5g/)
+
+4. Emphasizing your expertise in the speaker bio section: <br />
+[Taking a Multilingual Conversational Engine to Production: Theory to Reality](https://in.pycon.org/cfp/pycon-india-2023/proposals/taking-a-multilingual-conversational-engine-to-production-theory-to-reality~ejRQv/) <br />
+[Amaze your friends by painting images on Google Sheets using Python](https://in.pycon.org/cfp/pycon-india-2023/proposals/amaze-your-friends-by-painting-images-on-google-sheets-using-python~eER80/)
+
+5. Video showcasing presentation style and link to proposal: <br />
+[Securing your python applications and network using Zero Trust Security](https://in.pycon.org/cfp/pycon-india-2023/proposals/securing-your-python-applications-and-network-using-zero-trust-security~eg1kb/) <br />
+[Accelerating testing cycles of embedded devices using Python](https://in.pycon.org/cfp/pycon-india-2023/proposals/accelerating-testing-cycles-of-embedded-devices-using-python~bqxRp/)
+
+### Other links:
+
+- [Amaze your friends by painting images on Google Sheets using Python](https://in.pycon.org/cfp/pycon-india-2023/proposals/amaze-your-friends-by-painting-images-on-google-sheets-using-python~eER80/) - expertise
+- [OpenAI Whisper and it’s amazing power to do fine-tuning demonstrated on my mother tongue](https://in.pycon.org/cfp/pycon-india-2023/proposals/openai-whisper-and-its-amazing-power-to-do-fine-tuning-demonstrated-on-my-mother-tongue~eENWa/) - unique, tailored for diverse audience 
+
```

Comment:
If you need any help with submitting your proposal. Please register for the upcoming mentorship call here: https://bit.ly/mnts